lib/pure/os.nim

```diff
@@ -2321,8 +2321,8 @@ iterator walkDirRec*(dir: string,
   ## ---------------------   ---------------------------------------------
   ## followFilter            meaning
   ## ---------------------   ---------------------------------------------
-  ## ``pcDir``               follow real directories (default)
-  ## ``pcLinkToDir``         follow symbolic links to directories
+  ## `pcDir`               follow real directories (default)
+  ## `pcLinkToDir`         follow symbolic links to directories
```

Comment:
```suggestion
  ## `pcDir`                 follow real directories (default)
  ## `pcLinkToDir`           follow symbolic links to directories
```
Not sure what you're referring to in https://github.com/nim-lang/Nim/pull/17041#discussion_r576486835.

Reply:
thanks